skopt/optimizer/optimizer.py

```diff
@@ -94,6 +95,16 @@ class Optimizer(object):
     * `acq_optimizer_kwargs` [dict]:
         Additional arguments to be passed to the acquistion optimizer.
 
+    * `parallel_strategy` [string, default=`"cl_min"`]:
```

Comment:
It seems that it will give more flexibility to a user, so I feel it is good to have it in ask. Moved it there.